schema/datatypes/datatypes.fbs

```diff
@@ -55,8 +55,8 @@ enum FirmwareErrorCode: uint8 {
 /// Used for filtering tracker rotations in software
 enum FilteringType: uint8 {
 	NONE = 0,
-	INTERPOLATION = 1,
-	EXTRAPOLATION = 2
+	SMOOTHING = 1,
```

Comment:
The original naming is more clear. Smoothing is too vague. Same with prediction.

Reply:
I actually find the whole naming more vague. It didn’t even make sense.
We’re talking about the type of filtering, “interpolation” doesn’t mean much compared to “smoothing”.

Reply:
We’ve also had way too much user confusion concerning the old naming in GUI, and I don’t see a reason to not make the protocol match the GUI in this case.

Reply:
I'm not gonna bikeshed about this, since we just disagree and its just opinion. But its not worth changing the name here and breaking stuff. Just leave it as is for now.

Reply:
nvm we are breaking things anyway, whatever have it your way 🙃